### PR TITLE
0.5_initialize_project_structure_kolisnyk

### PR DESCRIPTION
**Pull request by kolesnicknick**
Create base structure for the BackEnd project

### CODE REVIEW

**Comment by IselDrew** [comment]
Rename 'seeds' folder to 'seeders'

**Comment by IselDrew** [comment]
Remove 'public' folder

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "outDir": "build",
+    "sourceMap": true,
+    "allowSyntheticDefaultImports": false,
+    "resolveJsonModule": true,
+    "experimentalDecorators": true
+  },
```
[the remark this answers is not in the document]
Comment (kolesnicknick) [reply tsconfig.json]: Added to tsconfig.json

File: src/server.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import * as express from 'express';
+import {Express, Request, Response, NextFunction} from "express-serve-static-core";
```
[the remark this answers is not in the document]
Comment (kolesnicknick) [reply src/server.ts]: Removed whole import

**Pull request merged**